vmconf.py

```diff
@@ -82,3 +82,7 @@
 
 # Prefix for data sent from Oppia to the Oppia-ml via JSON.
 XSSI_PREFIX = ')]}\'\n'
+
+# The regular expression used to identify whether a string contains float value.
+# The regex must match with regex that is stored in feconf.py file of Oppia.
+FLOAT_VERIFIER_REGEX = '^([-+]?\\d*\\.\\d+)$|^([-+]?\\d+\\.?\\d*e[-+]?\\d*)$'
```

Comment:
Add a whole bunch of tests for this -- at least 20 examples of floats written in different ways. Both for robustness and as a way to illustrate what's OK and not OK.